css/_user-picker.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * Styles errors and links in the user picker.
```

Comment:
Maybe don't call it the user picker here? This seems like the only place where it's called that?